wrapper/src/main/java/software/amazon/jdbc/util/monitoring/MonitorServiceImpl.java

```diff
@@ -64,7 +63,6 @@ public class MonitorServiceImpl implements MonitorService, EventSubscriber {
         TimeUnit.MINUTES.toNanos(15), TimeUnit.MINUTES.toNanos(3), recreateOnError);
 
     suppliers.put(ClusterTopologyMonitorImpl.class, () -> new CacheContainer(defaultSettings, Topology.class));
```

Comment:
A side question. Should we use implementation class name or an interface class name?

Reply:
I think implementation class, because we may at some point have multiple implementation classes for the same interface with each requiring different cache settings